lms/static/scripts/frontend_apps/components/AuthButton.tsx

```diff
@@ -1,19 +1,26 @@
 import { LabeledButton } from '@hypothesis/frontend-shared';
-
 import { useCallback, useEffect, useRef } from 'preact/hooks';
 
 import AuthWindow from '../utils/AuthWindow';
 
-/**
- * @typedef AuthButtonProps
- * @prop {string} authURL - Initial URL for the authorization popup. See `AuthWindow`.
- * @prop {string} authToken - Auth token between the LMS frontend and backend. See `AuthWindow`.
- * @prop {string} [label] - Custom label for the "Authorize" button
- * @prop {() => void} onAuthComplete - Callback invoked when the authorization flow completes.
- *   This does not guarantee that authorization was successful. Instead
- *   the caller should retry whatever operation triggered the authorization
- *   prompt and check whether it succeeds.
- */
+type AuthButtonProps = {
+  /** Initial URL for the authorization popup. See {@link AuthWindow}. */
+  authURL: string;
+  /** Auth token between the LMS frontend and backend. See {@link AuthWindow}. */
+  authToken: string;
+
+  /** Custom label for the "Authorize" button. */
```

Comment:
I don't mind the extra line breaks between props from here on, but am curious as to whether they mean something.

Reply:
Ah, this was accidental. I'm not sure if we should or shouldn't have these. I've removed the extra new-lines so at least this file is consistent. This is a formatting decision that Prettier doesn't handle for us unfortunately.